[2025-05-24T22:30:23.974+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-24T22:30:24.003+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T21:03:17.340189+00:00 [queued]>
[2025-05-24T22:30:24.012+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T21:03:17.340189+00:00 [queued]>
[2025-05-24T22:30:24.013+0000] {taskinstance.py:2303} INFO - Starting attempt 3 of 8
[2025-05-24T22:30:24.024+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): extract_sql_data> on 2025-05-24 21:03:17.340189+00:00
[2025-05-24T22:30:24.049+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'supplier_data_etl_pipeline', 'extract_sql_data', 'manual__2025-05-24T21:03:17.340189+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/supplier_dag.py', '--cfg-path', '/tmp/tmpri7_morl']
[2025-05-24T22:30:24.043+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1750) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-24T22:30:24.055+0000] {standard_task_runner.py:63} INFO - Started process 1755 to run task
[2025-05-24T22:30:24.055+0000] {standard_task_runner.py:91} INFO - Job 133: Subtask extract_sql_data
[2025-05-24T22:30:24.148+0000] {task_command.py:426} INFO - Running <TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T21:03:17.340189+00:00 [running]> on host a7d211f7b093
[2025-05-24T22:30:24.219+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='supplier_data_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_sql_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-24T21:03:17.340189+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-24T21:03:17.340189+00:00'
[2025-05-24T22:30:24.221+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-24T22:30:24.223+0000] {etl.py:77} INFO - connected to db source
[2025-05-24T22:30:24.248+0000] {etl.py:79} INFO - queried db source
[2025-05-24T22:30:24.249+0000] {etl.py:81} INFO - closed db connection
[2025-05-24T22:30:24.253+0000] {etl.py:83} INFO - db source pickled
[2025-05-24T22:30:24.254+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-24T22:30:24.254+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-24T22:30:24.261+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=supplier_data_etl_pipeline, task_id=extract_sql_data, execution_date=20250524T210317, start_date=20250524T223024, end_date=20250524T223024
[2025-05-24T22:30:24.279+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-24T22:30:24.299+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-24T22:30:24.300+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
